middlewares/http/middleware.go

```diff
@@ -239,3 +228,37 @@ func GetMiddleware(options *Options) (func(next http.Handler) http.Handler, erro
 
 	return middleware, nil
 }
+
+func getRouter(options *Options) (routers.Router, error) {
+	if (options.OpenapiBytes == nil || len(options.OpenapiBytes) == 0) && options.OpenapiSpecPath == "" {
```

Comment:
Use expressive boolean variables to make reading the logic easier

```suggestion
        hasBytes := (options.OpenapiBytes != nil && len(options.OpenapiBytes) > 0)
        hasSpecPath :=  options.OpenapiSpecPath != ""
        
	if !hasBytes && !hasSpecPath {
```